### PR TITLE
added device-info API

### PR DESCRIPTION
**Pull request by gferraro**

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -5,3 +5,7 @@
 [[constraint]]
   name = "github.com/gobuffalo/packr"
   version = "1.12.0"
+
+[[constraint]]
+  name = "github.com/gferraro/go-api"
```
Comment (mjs) [inline Gopkg.toml]: I think you mean the TheCacophonyProject repo, not this one. Get any go-api changes landed before merging this.

File: api/api.go [hunk 2]
```diff
@@ -49,6 +50,20 @@ func NewAPI(cptvDir string) *ManagementAPI {
 	}
 }
 
+// GetDeviceInfo returns information about this device
+func (api *ManagementAPI) GetDeviceInfo(w http.ResponseWriter, r *http.Request) {
+	log.Println("get device info")
```
Comment (mjs) [inline api/api.go]: Don't need this

File: api/api.go [hunk 3]
```diff
@@ -49,6 +50,20 @@ func NewAPI(cptvDir string) *ManagementAPI {
...
+	config, err := goAPI.LoadConfig()
+	if err != nil {
+		fmt.Printf("error is %v", err)
```
Comment (mjs) [inline api/api.go]: make this:
```go
log.Printf("/device-info failed: %v", err)
```

File: api/api.go [hunk 4]
```diff
@@ -30,6 +30,7 @@ import (
 	"strconv"
 
 	signalstrength "github.com/TheCacophonyProject/management-interface/signal-strength"
+	goAPI "github.com/gferraro/go-api"
```
Comment (mjs) [inline api/api.go]: In idomatic Go, import aliases should just be lower case - so `goapi` in this case

File: Gopkg.toml (hunk 1 above)
Comment (gferraro) [reply Gopkg.toml]: yep just there for me to test with at the moment

File: api/api.go [hunk 5]
```diff
@@ -30,6 +30,7 @@ import (
 	"strconv"
 
 	signalstrength "github.com/TheCacophonyProject/management-interface/signal-strength"
+	goapi "github.com/gferraro/go-api"
```
Comment (mjs) [inline api/api.go]: you still have the wrong import path here (which is causing the build to fail)

**Pull request merged**